moovingon_ai/README.md

```diff
@@ -0,0 +1,47 @@
+# moovingon.ai
+
+## Overview
+[moovingon.ai][1] is a platform for cloud operations and NOC management. It consolidates alerts across your observability suite and associates them with automated runbooks for alert and incident remediation. With this integration, you can utilize the power of Datadog and moovingon.ai for efficient, automated incident management.
+
+moovingon.ai uses Datadog's monitors, logs, and event data for alert correlation and aggregation.
+Key features of this integration include:
+
+1. **Focused Alert Management**: Bring all your Datadog alerts together in moovinon.ai easy-to-view dashboard, giving you simple control and clear visibility.
+2. **Comprehensive incident management**: All remediation actions performed in moovingon.ai are sent to Datadog as events for compliance and remediation clarity
+3. **Extensive Analysis**: Utilize the analytics provided by moovingon.ai to derive insights from Datadog alerts. This assists in proactive decision-making and trend analysis.
+
+## Setup
+
+### Installation
+
+1. Click on **Connect Accounts** in order to login into moovingon.ai.
+2. Enter a name for the datadog integration and **Submit**.
+3. Proceed to the Datadog OAuth2 screen and click on the **authorize** button.
```

Comment:
Same question as above: Is this in lowercase in the UI?

Reply:
Nope, fixed